src/Compilers/CSharp/Portable/Parser/Lexer_StringLiteral.cs

```diff
@@ -457,11 +474,16 @@ private void ScanFormatSpecifier()
                         {
                             TrySetUnrecoverableError(_lexer.MakeError(pos, 1, ErrorCode.ERR_EscapedCurly, ch));
                         }
+                        else if (ch is '"' or ':')
+                        {
+                            TrySetUnrecoverableError(_lexer.MakeError(pos, 1, ErrorCode.ERR_UnexpectedCharacter, ch));
+                        }
                     }
                     else if (ch == '"')
                     {
                         if (_isVerbatim && _lexer.TextWindow.PeekChar(1) == '"')
                         {
+                            TrySetUnrecoverableError(_lexer.MakeError(_lexer.TextWindow.Position, 1, ErrorCode.ERR_UnexpectedCharacter, "\""));
```

Comment:
why was this change made?

Reply:
the same as for ':' - '"' is not allowed
But we could ignore the spec here, too.